apps/bajour/src/components/quote/bajour-quote.tsx

```diff
@@ -0,0 +1,121 @@
+import {css, styled, Theme, useTheme} from '@mui/material'
+import {BuilderQuoteBlockProps, useWebsiteBuilder} from '@wepublish/website'
+import {useMemo} from 'react'
+
+export const BajourQuoteBlockWrapper = styled('blockquote')<{withImage: boolean}>`
+  font-style: italic;
+  position: relative;
+  margin: 0;
+  padding-left: calc((100% / 48));
+  padding-right: calc((100% / 48));
+
+  ${({theme, withImage}) =>
+    withImage &&
+    css`
+      ${theme.breakpoints.up('sm')} {
+        flex-direction: row;
+      }
+    `};
+
+  ${({theme}) => theme.breakpoints.up('sm')} {
+    max-width: calc((100% / 48) * 2 + ${({theme}) => theme.spacing(85)});
+    margin-left: 0;
+  }
+
+  ${({theme}) => theme.breakpoints.up('md')} {
+    margin: 0;
+    margin-left: ${({theme}) => `-${theme.spacing(8)}`};
+    max-width: calc((100% / 12) * 4 + ${({theme}) => theme.spacing(60)});
+  }
+
+  @media (min-width: 1440px) {
+    // breakpoint needs to be custom
```

Comment:
Explain why please

Reply:
that's because the component is pushed a bit to the left via negative margin, at least this is what I understood that is required looking at the designs 
<img width="703" alt="Screenshot 2024-05-23 at 14 10 20" src="https://github.com/wepublish/wepublish/assets/24550916/a7ef09da-1d85-41c9-a025-15f5d89680ec">

Reply:
I still don't understand why the breakpoint needs to be custom